test/contains.js

```diff
@@ -28,6 +28,64 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `contains` method', function() {
+    function Empty() {}
+    Empty.prototype.contains = R.always(false);
+
+    function List(head, tail) {
+      this.head = head;
+      this.tail = tail;
+    }
+    List.prototype.contains = function(x) {
+      return this.head === x || this.tail.contains(x);
+    };
+
+    var list = new List('foo', new List('bar', new List('baz', new Empty())));
+
+    assert.strictEqual(R.contains('foo', list), true);
+    assert.strictEqual(R.contains('bar', list), true);
+    assert.strictEqual(R.contains('baz', list), true);
+    assert.strictEqual(R.contains('quux', list), false);
+  });
+
+  it('dispatches to `indexOf` method', function() {
```

Comment:
If this PR gets merged this test _has_ to be there. It is written in the documentation that the function dispatches to `indexOf`. It is in the title of the PR. It is the entire point of the PR. It changes the definition of _contains_ to expose what would otherwise be an implementation detail.

We can not deliberately add a certain behavior to a function _and_ document it without also testing it. Otherwise someone might change this in half a year for some reason and break both the documentation and a lot of peoples code without any tests failing.